### PR TITLE
issue 526.9.1: implement swagger interface, add documentation

### PR DESCRIPTION
**Pull request by vasiliy-arangodb**

### CODE REVIEW

File: Documentation/Books/Manual/Analyzers/README.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
+# Analyzers powered by IResearch
+
+# Background
+
+A concept of value "analysis" that is meant to break up a given value into
```
Comment (graetzer) [inline Documentation/Books/Manual/Analyzers/README.md]: this sentence sounds a bit wonky

File: Documentation/Books/Manual/Analyzers/README.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+The REST API is accessible via the `/_api/analyzer` endpoint URL callable via
+HTTP requests.
+
+The *create* operation is accessible via the *POST* method on the URL:
+```
+/_api/analyzer
+```
+With the Analyzer configuration passed via the body as an object with
+attributes: <br/>
```
Comment (graetzer) [inline Documentation/Books/Manual/Analyzers/README.md]: shouldn't you just redirect to the relevant section in the http book ?

Comment (vasiliy-arangodb) [reply Documentation/Books/Manual/Analyzers/README.md]: How would you do that, aren't those sections autogenerated, i.e. how would you know the name of the file beforehand?
Additionally the explanations here are more verbose and explain both the JavaScript and REST APIs

File: Documentation/Books/Manual/Analyzers/README.md (hunk 1 above)
Comment (vasiliy-arangodb) [reply Documentation/Books/Manual/Analyzers/README.md]: that's what you get for copy-pasting from another place in the documentation
done

**Comment by vasiliy-arangodb** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/4025/

File: Documentation/Books/Manual/Analyzers/README.md (hunk 2 above)
Comment (dothebart) [reply Documentation/Books/Manual/Analyzers/README.md]: You have to link to the `.html`-file.

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/4098/

**Pull request merged**